src/commander.ts

```diff
@@ -272,4 +272,47 @@ export class Commander {
         });
     }
 
+    static validateDependencyChain (parser: Parser, writeStreams: WriteStreams): boolean {
```

Comment:
Isn't this a complete dublicate of the code in https://github.com/firecow/gitlab-ci-local/blob/master/src/executor.ts